vitrivr-engine-core/src/main/kotlin/org/vitrivr/engine/core/model/content/impl/memory/InMemoryMeshContent.kt

```diff
@@ -12,4 +13,6 @@ import org.vitrivr.engine.core.model.mesh.Model3D
  * @author Luca Rossetto.
  * @version 1.0.0
  */
-data class InMemoryMeshContent(override val content: Model3D) : Model3DContent
+data class InMemoryMeshContent(override val content: Model3D) : Model3DContent {
+    override val id: UUID = UUID.randomUUID()
```

Comment:
I would add this to the constructor. There might be use-cases, where one wants to define this explicitly. You can also use default values in the constructor.